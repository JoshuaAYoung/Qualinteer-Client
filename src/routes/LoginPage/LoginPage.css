.loginInput {
  width: 100%;
  height: 3.5rem;
  font-size: 1.7rem;
  padding-left: 1rem;
  color: black;
  padding-bottom: .2em;
  }

.tallerInput {
margin-bottom: .6rem;
}

.registerLink {
  font-weight: 600;
  color: black;
}

.loginForm {
  margin-top: 1.8rem;
  margin-bottom: 6rem;
}

.demoInstructions {
  margin: 5.5rem 0 3rem;
  line-height: 2.3rem;
}

.demoLogin {
  margin-bottom: 1rem;
}

.showLoginButton, .hideLoginButton {
  width: 4rem;
  height: 3.5rem;
  border-radius: 0;
  position: absolute;
  z-index: 1;
  right: 0;
  background-repeat: no-repeat;
  background-size: 1.8rem;
  background-position: center;
}

.showLoginButton {
  background-image: url("/assets/show-password.svg");
}

.hideLoginButton {
  background-image: url("/assets/hide-password.svg");
}

.hideLoginButton:hover, .showLoginButton:hover {
  background-color: #F0667B
}

.loginButton {
  margin-top: 1rem;
}

.registerInstructions {
  margin-top: 2rem;
  font-size: 1.4rem;
}

@media only screen and (min-width: 1024px) {
  .loginFormContainer {
    padding: 0 20rem;
  }
  .demoInstructions {
    margin-top: 5rem;
  }
  .registerInstructions {
    margin-top: 4rem;
  }
}