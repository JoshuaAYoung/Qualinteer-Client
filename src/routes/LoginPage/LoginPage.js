import React from 'react';
import LoginForm from '../../components/LoginForm/LoginForm';
import './LoginPage.css';
import ScrollToTopOnMount from '../../utilities/ScrollToTopOnMount';

export default function LoginPage(props) {
  return (
    <div className="loginFormContainer">
      <ScrollToTopOnMount />
      <LoginForm history={props.history} />
    </div>
  );
}
