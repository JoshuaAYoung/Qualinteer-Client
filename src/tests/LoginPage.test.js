import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoginPage from '../routes/LoginPage/LoginPage';

describe('LoginPage Component', () => {
  it('LoginPage Smoke Test', () => {
    const div = document.createElement('div');

    ReactDOM.render(
      <BrowserRouter>
        <LoginPage />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it('LoginPage Snapshot Test', () => {
    const wrapper = shallow(<LoginPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
