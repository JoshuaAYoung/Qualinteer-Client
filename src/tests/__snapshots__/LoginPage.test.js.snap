// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage Component LoginPage Snapshot Test 1`] = `
<div
  className="loginFormContainer"
>
  <ScrollToTopOnMount />
  <LoginForm />
</div>
`;
